src/celeritas/em/xs/LivermorePEMicroXsCalculator.hh

```diff
@@ -28,14 +28,15 @@ class LivermorePEMicroXsCalculator
   public:
     //!@{
     //! \name Type aliases
-    using XsUnits = LivermoreSubshell::XsUnits;
+    using ParamsRef = LivermorePERef;
     using Energy = Quantity<LivermoreSubshell::EnergyUnits>;
+    using XsUnits = LivermoreSubshell::XsUnits;
```

Comment:
Following the suggestion below, let's define the actual "type" of a real value wrapped in a `Barn` unit:
```suggestion
    using Xs = Quantity<LivermoreSubshell::XsUnits>;
```

Reply:
Using BarnXs.